core/server/data/migrations/versions/3.12/01-add-identity-permission.js

```diff
@@ -0,0 +1,56 @@
+const ObjectId = require('bson-objectid');
+const common = require('../../../../lib/common');
+
+module.exports.config = {
+    transaction: true
+};
+
+module.exports.up = async (options) => {
+    const connection = options.transacting;
+
+    const existingIdentityPermission = await connection('permissions').where({
+        action_type: 'read',
+        object_type: 'identity'
+    }).first();
+
+    if (existingIdentityPermission) {
+        common.logging.warn('Adding permission for read:identity');
```

Comment:
@allouis Think the warn logs here and for down migration below needs to change as they log the same thing as normal case right now which is not much useful 🤔 

```
Permission for read:identity already exists/doesn't exist
```

Reply:
FWIW most our migrations do this and use info/warn to differentiate between success. but will update

Reply:
@allouis Oh i was probably remembering an old one in that case, please ignore this in that case, nothing to bother with :)